src/board.hpp

```diff
@@ -3,13 +3,25 @@
 
 #include "coords.hpp"
 #include "enums.hpp"
+#include <vector>
+#include <optional>
+#include <array>
+
 
 namespace TicTacToe {
+    using BoardLine = std::array<Coords, BOARD_WIDTH>;
     class Board {
+    private:
+        std::optional <Symbol> board [BOARD_WIDTH][BOARD_WIDTH];
+        ;
     public:
-        std::optional <Symbol> fieldContent(Coords coords);
+        std::vector<BoardLine> winningPossibilities();
+
+        std::vector<BoardLine> testTaMetoda();
```

Comment:
- clean this method up - make it clear that this is a temp hack made to be able to test a private method
  - it'd be best though - and ultimately we *will* have to do that anyway - if we found a better way so we don't have to do that further on for every single private method

- change `winningPossibilities` to be private

Reply:
btw would you like to start using github's issue tracker to organize our next steps so that we don't get stuck with "oh idk what am I supposed to do next"?